.github/workflows/continuous-integration-cypress.yml

```diff
@@ -0,0 +1,40 @@
+name: Cypress tests
+
+on:
+  workflow_run:
+    workflows: ["Deploy to environment (CIP)"]
+    branches: [ main ]
+    types:
+      - completed
+
+env:
+  NODE_VERSION: 18.x
+
+jobs:
+  cypress-tests:
+    if: ${{ github.event.workflow_run.conclusion == 'success' }}
+    environment: ${{ github.event.workflow_run.environment }}
```

Comment:
This is something I was struggling to find for the transfer project. Just so I am clear "github.event.workflow_run.environment" will give you the environment that was used in the previous workflow that triggered this workflow?

Reply:
I believe this to be the case from my research but I haven't been able to confirm it yet. (this will be the first time I have tried to use it). According to the Github docs, `github.event.workflow_run` should include the GitHub context for that workflow, so in theory it should include the environment too. If this doesn't work after all, we could try `github.event.workflow_run.inputs.environment`. Here's hoping that this is possible

Reply:
Sounds good 🤞 
I think I tried and failed to use - github.event.workflow_run.inputs.environment